README.md

```diff
@@ -486,21 +512,48 @@ await client.insert({
 
 ### Insert with stream input ("raw" formats like CSV* / TabSeparated* / CustomSeparated\*, flowing stream)
 
+Raw format flowing stream insertion should be similar to JSON* one,
+with an exception that the Readable should have `objectMode` set to `false`.
+
 ```ts
-const stream = new Stream.Readable({
-  objectMode: false, // required for "raw" family formats
-  read() {
-    /* stub */
-  },
-})
-// ... your (async) code pushing the values into the stream...
-await client.insert({
-  table: tableName,
-  values: stream,
-  format: 'TabSeparated', // or any other desired "raw" format
-})
-// close the stream when finished by pushing a null value there
-stream.push(null)
+;(async () => {
+  const stream = new Stream.Readable({
+    objectMode: false, // required for "raw" family formats
+    read() {
+      //
+    },
+  })
+  // note that we do not await the promise yet
+  const insertPromise = client.insert({
+    table: 'mytable',
+    values: stream,
+    format: 'CSV',
+  }).then(() => console.info('Data ingestion is finished'))
+
+  // interval just for example purposes
+  const timer = setInterval(fetchAndStore(stream), 5000)
+
+  async function cleanup() {
+    clearInterval(timer)
+    // close the stream
+    stream.push(null)
+    // when the stream is closed, the insert stream can be awaited
+    await insertPromise
+    await client.close()
+    process.exit(0)
+  }
+  process.on('SIGINT', cleanup)
+  process.on('SIGTERM', cleanup)
+})()
+
+function fetchAndStore(stream: Stream.Readable) {
```

Comment:
Maybe it's better to put this code in examples? I'd keep README short and provide code snippets focused on client API

Reply:
Yep, thought that as well. Shall maybe we do a follow-up PR to extract _all_ the examples from the README, merging this one for now?